common4j/src/main/com/microsoft/identity/common/java/controllers/results/INativeAuthCommandResult.kt

```diff
@@ -29,10 +29,13 @@ import com.microsoft.identity.common.java.logging.DiagnosticContext
  * INativeAuthCommandResult interface defines the base class for errors used in Native Auth.
  */
 interface INativeAuthCommandResult {
-    data class Redirect(val correlationId: String = DiagnosticContext.INSTANCE.threadCorrelationId) :
-        SignInStartCommandResult, SignInWithSLTCommandResult, SignInSubmitCodeCommandResult,
-        SignInResendCodeCommandResult, SignInSubmitPasswordCommandResult,
-        SignUpStartCommandResult, SignUpSubmitCodeCommandResult,
+    data class Redirect(
+        override val error: String = "browser_required",
```

Comment:
What other errors other than "browser_required" is possible for `Redirect` ?

Reply:
I personally think Redirect = browser_rquired, but using different naming.

Reply:
If "browser_required" is the only possible value passed to the constructor of Redirect class then it should not be part of the constructor. We should take it as a constant value and use that constant as a parameter for calling Error class.

Reply:
yes, that's better but the constants should have name in all caps and their name should reflect the value contained. Their names could be `BROWSER_REQUIRED_ERROR` and `BROWSER_REQUIRED_ERROR_DESCRIPTION`